src/Files.App/Commands/Manager/CommandManager.cs

```diff
@@ -168,6 +174,14 @@ public CommandManager()
 				.Append(new NoneCommand())
 				.ToImmutableDictionary(command => command.Code);
 
+			_ModifiableDeleteItem = new ModifiableCommand(commands[CommandCodes.DeleteItem], new() {
+				{ KeyModifiers.Shift,  commands[CommandCodes.DeleteItemPermanently] }
+			});
+
+			_ModifiablePasteItem = new ModifiableCommand(commands[CommandCodes.PasteItem], new() {
+				{ KeyModifiers.Shift,  commands[CommandCodes.PasteItemToSelection] }
+			});
+
```

Comment:
I am not a fan at all of instanciating those methods here. They should at least be done inside of a dedicated method, possibly called by the constructor. I'd go as far as create a smaller class contained by the Command Manager to store the modifiable commands? If we get more modifiables here, it's gonna be really crowded soon.

Reply:
The instantiating of commands has been carved out into a separate method. Also, the management of ModifiableCommand itself has been carved out into a new class.